src/components/forms/OnboardingProfessionalForm.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
```

Comment:
Regarding the file name, as we have a folder called `forms`, it's kind of redundant to have the suffix `Form` in all components inside this folder. I think we should remove it